### ISSUE TITLE
0.5.1 not available on pypi.org

### ISSUE THREAD

**Issue by lent1** [Patreon/patreon-python#30]
I'm trying to use patreon api with python.  
patreon-python is a great library but v0.5.1, which use latest api endpoints, is not available on pypi.org.  

Do you plan to publish this version on pypi?

**Comment by jeffheaton** [comment]
Agree, just trying to follow the instructions on README you will get this error:
```
AttributeError: 'API' object has no attribute 'get_identity'
```
This is due to the fact that get_identity is not a part of the version currently on pipy.

**Comment by jenyeeiam** [comment]
Any resolution to this??

**Comment by nstephenh** [comment]
I would also like to see this updated.

**Comment by kbyatnal** [comment]
@phildini are you still the main contributor? Could you take a look at this?

**Comment by kbyatnal** [comment]
In the meantime, you can run (if you're using `pipenv`) `pipenv install -e git+https://github.com/Patreon/patreon-python.git@master#egg=patreon` to get the latest version. Haven't tried for `pip` but I imagine it's very similar.

**Comment by Ganonmaster** [comment]
I just ran into this issue myself, and having the correct version up on PyPI would have saved me some time. Since this repository has seen virtually no changes for over a year, it's probably safe to push the update?

Also; should we be concerned that the primary contributors to the repository are no longer with the company?

**Comment by DrLarck** [comment]
> 
> 
> In the meantime, you can run (if you're using `pipenv`) `pipenv install -e git+https://github.com/Patreon/patreon-python.git@master#egg=patreon` to get the latest version. Haven't tried for `pip` but I imagine it's very similar.

Inspired by this method, I've installed this repo as a python module using pip by typing the following command : 
```
pip install git+https://github.com/Patreon/patreon-python.git@master#egg=patreon
```

**Comment by AznStevy** [comment]
The version for python3.8 (which is what the rest of my project is in) is 0.3.1. Is there no way to update to 0.5.0+?

**Comment by themagic314** [comment]
`pip install --upgrade https://github.com/Patreon/patreon-python/tarball/master`

**Comment by geerlingguy** [comment]
Coming here years after the fact... it feels strange none of the Patreon API libraries have been updated in 6 or so years :(

I thought it would be easy to write a little script to grab patron names by tier but am finding it to be frustrating, probably easier to just interact with the API endpoints directly instead of using this library!
